example/python/prepare.sh

```diff
@@ -5,9 +5,9 @@ cd $(dirname $0)
 # generate iroha lib
 CURDIR="$(cd "$(dirname "$0")"; pwd)"
 IROHA_HOME="$(dirname $(dirname "${CURDIR}"))"
-cmake -H$IROHA_HOME -Bbuild -DSWIG_PYTHON=ON -DSHARED_MODEL_DISABLE_COMPATIBILITY=ON;
+cmake -H$IROHA_HOME -Bbuild -DSWIG_PYTHON=ON -DSHARED_MODEL_DISABLE_COMPATIBILITY=ON -DSUPPORT_PYTHON2=OFF;
```

Comment:
Better use: `python -c "print('OFF' if __import__('sys').version_info[:][0] == 3 else 'ON')"`

Reply:
`python` is by default 2, to run the 3rd one you have to do it explicitly: `python3`